### PR TITLE
PS-9000 Release notes for PS-8.0.35-27

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0.35-27.md
modified:   docs/release-notes/8.0.35-27.md
modified:   docs/release-notes/release-notes_index.md
modified:   mkdocs-base.yml
modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0.35-27.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+* [PS-7806](https://jira.percona.com/browse/PS-7806): Tables with column compression broke asynchronous replication.
+
+* [PS-8671](https://jira.percona.com/browse/PS-8671): The `innodb.ddl_kill` test failed on Jenkins.
+
+* [PS-8741](https://jira.percona.com/browse/PS-8741): MySQL 8.0.33 failed in Debug mode while running `innodb.ddl_crash_alter_partition`.
+
+* [PS-8879](https://jira.percona.com/browse/PS-8879): Received an out of memory error when running `ALTER TABLE ... COLUMN_FORMAT COMPRESSED;`.
+
+* [PS-8965](https://jira.percona.com/browse/PS-8965): Fix for the procession of deleted keys by `keyring_vault` API when Vault_version_v2 was used.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.35-27.md]: In some situations, the keyring_vault retrieved records for already deleted keys with empty 'data' and 'deletion_time' set. Ensured that deleted keys were properly processed.

File: docs/release-notes/8.0.35-27.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* The `--character-set-client-handshake` server variable is deprecated. The usage of this variable results in an warning. This variable will be removed in future releases.
+* `INFORMATION_SCHEMA.PROCESSLIST` is deprecated. The implementation of `SHOW PROCESSLIST` which uses that table is also deprecated. Use the Performance Schema implementation of `SHOW PROCESSLIST` instead. `performance_schema_show_processlist` is also deprecated.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.35 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html).
+
+## Bug fixes
+
+* [PS-7806](https://jira.percona.com/browse/PS-7806): Tables with column compression broke asynchronous replication.
+
+* [PS-8671](https://jira.percona.com/browse/PS-8671): The `innodb.ddl_kill` test failed on Jenkins.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.35-27.md]: An ocassional failure of the `innodb.ddl_kill` test on Jenkins.

File: docs/release-notes/8.0.35-27.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.35-27.md]: fixed

File: docs/release-notes/8.0.35-27.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.35-27.md]: fixed

File: docs/release-notes/8.0.35-27.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+* [PS-8849](https://jira.percona.com/browse/PS-8849): The server exited with the `Assertion failure: row0mysql.cc:218:len < 256 * 256 thread`.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.35 and included in Percona Server for MySQL are the following:
+
+* Upgraded the linked OpenSSL library to OpenSSL 3.0.10.
+* Removed the printed query string limit to display the characters for a detected deadlock section of the engine status log.
+* Fixed a procession of single-character tokens by an FTS parser plugin.
+
+### Deprecations and removals
```
Comment (patrickbirch) [inline docs/release-notes/8.0.35-27.md]: Maybe call this section 
Deprecations
I didn't see any removals.

**Review by patrickbirch — APPROVED** [review]
Good - except for one thing

File: docs/release-notes/8.0.35-27.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.35-27.md]: fixed

File: docs/release-notes/8.0.35-27.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* [PS-8741](https://jira.percona.com/browse/PS-8741): MySQL 8.0.33 failed in Debug mode while running `innodb.ddl_crash_alter_partition`.
+
+* [PS-8879](https://jira.percona.com/browse/PS-8879): Received an out of memory error when running `ALTER TABLE ... COLUMN_FORMAT COMPRESSED;`.
+
+* [PS-8965](https://jira.percona.com/browse/PS-8965): In some situations, the `keyring_vault` retrieved records for already deleted keys with empty `data` and `deletion_time` set. Ensured that deleted keys were properly processed.
+
+* [PS-8973](https://jira.percona.com/browse/PS-8973): The execution of `audit_log_filter.status_var_log_size` failed if it was run multiple times.
+
+* [PS-8975](https://jira.percona.com/browse/PS-8975): When userstat was enabled, concurrent execution of queries to tables, including queries to `I_S.TABLE/INDEX_STATISTICS` and `FLUSH PRIVILEGES` commands caused deadlocks between internal locks of the Percona Server. Statements in the existing connections were stalled, and new connections to the server failed (Thanks to Pierre POMES for reporting this issue.)
+
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.35-27.md:55]: There are also
https://perconadev.atlassian.net/browse/PS-8927
and
https://perconadev.atlassian.net/browse/PS-9011
in the https://perconadev.atlassian.net/browse/RM-1292 blocker list.
Any reason they are not in this list

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.35-27.md:55]: added

File: docs/release-notes/8.0.35-27.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* [PS-8927](https://perconadev.atlassian.net/browse/PS-8927): The `CREATE TABLE LIKE` option didn't write the charset on binary log if it was created from a temporary table.
+
+* [PS-9011](https://perconadev.atlassian.net/browse/PS-9011): Downstream INSTANT algo <= 8028 on table in mysql schema led to corruption post upgrade.
+
+* []():
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.35-27.md]: Is this a formatting marker or just a bug template?

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.35-27.md]: the bug has been fixed

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**